soc/arm/arm/musca_b1/Kconfig.series

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config SOC_SERIES_MUSCA_B1
-	bool "arm v2m MUSCA B1 MCU Series"
+	bool "ARM v2m MUSCA B1 MCU Series"
```

Comment:
Has to be Arm, then, to comply with the newest standard.

Reply:
Thanks for this information.  Do you have an URL handy that I can put in the commit msg?

So this will require changing the other Arm SoCs as well, so I'll update the PR accordingly.

Reply:
https://www.arm.com/